src/Middleware/RequestThrottling/src/RequestThrottlingOptions.cs

```diff
@@ -22,5 +24,13 @@ public class RequestThrottlingOptions
         /// Defaults to 5000 queued requests.
         /// </summary>
         public int RequestQueueLimit { get; set; } = 5000;
+
+        /// <summary>
+        /// 
```

Comment:
We still need add a doc comment for this though.

Reply:
maybe "Optional callback; processes requests rejected by the middleware."

Reply:
Everything has defaults, so I don't think we should bother pointing out it's optional. How about this?

"A <see cref="RequestDelegate"/> that handles requests rejected by this middleware. If it doesn't modify the response, an empty 503 response will be written."

Reply:
Sorry to make you iterate so much @Kahbazi. @DylanDmitri I'm fine with you merging this as is and fixing up the doc comment later.

Reply:
no, it's ok. I can change it again when you decided what you want it to be. I think maybe "A callback that handles rejected request by this middleware." is good enough since 503 status code is now part  of the middleware and not OnRejected.

Reply:
I do like making it clear that it's a RequestDelegate that can modify the response that is chooses to. I just realized my last comment wasn't formatted properly so you couldn't see the `<see cref="RequestDelegate"/>`'s. It was meant to be:

`"A <see cref="RequestDelegate"/> that handles requests rejected by this middleware. If it doesn't modify the response, an empty 503 response will be written."`

@DylanDmitri @Kahbazi Does this seem reasonable to you?

Reply:
Stephen's description is better and more professional